src/risedevtool/src/task/meta_node_service.rs

```diff
@@ -245,6 +260,18 @@ impl Task for MetaNodeService {
         cmd.arg("--config-path")
             .arg(Path::new(&prefix_config).join("risingwave.toml"));
 
+        if let MetaBackend::Env = self.config.meta_backend {
+            if is_env_set("RISEDEV_CLEAN_START") {
```

Comment:
It might be a little confusing that `RISEDEV_CLEAN_START` only works with `meta-backend: env`

Reply:
It's not meant to be set by user directly but when running `risedev ci-start`.

The basic idea is that, I don't want to hack the procedure of `risedev clean-data` to involve such operation, so it's postponed to the next startup. It only applies to `ci-start` because

- the main motivation is CI as described in #19527
- for `ci-start` we are sure that we want a fresh cluster